source/jormungandr/jormungandr/interfaces/v1/StatedResource.py

```diff
@@ -61,3 +62,15 @@ def wrapper(*args, **kwargs):
                 register_used_coverages([region])
             return result
         return wrapper
+
+    def dispatch_request(self, *args, **kwargs):
```

Comment:
This is to deactivate decorators for OPTIONS method.
The decorators are actually expect the GET result in input